### DISCUSSION TITLE
QueryException for missing language_lines table during package installation

### DISCUSSION THREAD

**Discussion by emilsrits** [spatie/laravel-translation-loader#135] [Bugs]
Laravel: 8.54
spatie/laravel-translation-loader: 2.7

During package installation an exception gets thrown about missing `language_lines` table before we can even publish migration files:
```
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'database.language_lines' doesn't exist (SQL: select * from `language_lines` where `group` = *)
```
This breaks all artisan commands. 

Workaround is to manually get the migration file from repository and migrate it before installing this package.
This also means every time you make a fresh setup of a project using this package you need to either have already imported database which has `language_lines` table or remove this package, comment out provider, migrate and then install it again.

Related to: https://github.com/spatie/laravel-translation-loader/discussions/130, https://github.com/spatie/laravel-translation-loader/discussions/123

**Comment by BorisKM** [comment]
The same problem..

[a comment by an automated account is not shown]
